framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
I already described here https://github.com/qooxdoo/qooxdoo/pull/9275#issuecomment-275117335 what the use case would be: If you have multiple buttons in different positions regular edit menu AND context menu both having a button for eg 'CTRL-V'